x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/SecurityFeatureSetUsage.java

```diff
@@ -39,6 +43,10 @@ public SecurityFeatureSetUsage(StreamInput in) throws IOException {
         realmsUsage = in.readMap();
         rolesStoreUsage = in.readMap();
         sslUsage = in.readMap();
+        if (in.getVersion().onOrAfter(Version.V_7_1_0)) {
```

Comment:
If I got it right should be `Version.CURRENT` and do the dance 😢 : commit to master, change to `Version.V_7_1_0` in the backport commit, disable bwc tests, commit to 7.x, wait for green intake, then change to `Version.V_7_1_0` in the master, and enable bwc tests . If you learn of an easier method lemme know please! I'll admit it, in the past I cut a few corners on this...